src/Controller/FightersController.php

```diff
@@ -14,11 +14,37 @@ public function index(): string
         return $this->twig->render('Fighters/fighterslist.html.twig', ['fighters' => $fighters]);
     }
 
-    public function pick(int $id): string
+    public function pick(): string
     {
+        // set fighters
+
         $fighterManager = new FighterManager();
-        $fighter = $fighterManager->selectOneById($id);
+        $fighters = $fighterManager->selectAll('id');
+
+        if (isset($_SESSION['player1']) && isset($_SESSION['player2'])) {
+            header('Location: /fight/attack');
+        }
+
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            if (!isset($_SESSION['player1'])) {
+                $_SESSION['player1'] = $fighterManager->selectOneById($_POST['selected_fighter']);
+                header('Location: /fight/pick');
+            } elseif (!isset($_SESSION['player2'])) {
+                $_SESSION['player2'] = $fighterManager->selectOneById($_POST['selected_fighter']);
```

Comment:
after this line, put:

    $_SESSION['currentAttacker'] = $_SESSION['player1'];